### ISSUE TITLE
order of subsections in "documentation"

### ISSUE THREAD

**Issue by jwaldmann** [haskell-infra/hl#57]
I think the primary resource should come first: the language definition. It is the only authoritative document on Haskell, right?  Since the reference is short, it can easily be moved to top of the page. All other resources (tutorials, etc.) are secondary. (Well, compiler-dependent language extensions are somewhere in between.)

Well maybe it shows that I've done too much teaching. First thing I have my students do, regularly, is find and bookmark authoritative definitions of programming languages they are using.

**Comment by chrisdone** [comment]
Your students are lucky that they are supported by a teacher. ;-) The first thing newbies ask is what Haskell book to get for learning, only later does having the report as a reference. I made this page so obviously I think the order of this content is appropriate. Deferring for input from other people.

**Comment by gbaz** [comment]
Yeah I strongly disagree here. The report is really not the first thing for a general audience.
